search-commons/src/main/java/no/unit/nva/search/ElasticSearchHighLevelRestClient.java

```diff
@@ -134,14 +140,29 @@ public void removeDocumentFromIndex(String identifier) throws SearchException {
 
     public AcknowledgedResponse prepareIndexForBatchInsert() throws IOException {
         Settings indexSettings = Settings.builder()
-                                     .put(ELASTIC_SEARCH_INDEX_REFRESH_INTERVAL, FIFTEEN_MINUTES)
-                                     .put(ELASTIC_SEARCH_NUMBER_OF_REPLICAS, 0)
-                                     .build();
+            .put(ELASTIC_SEARCH_INDEX_REFRESH_INTERVAL, FIFTEEN_MINUTES)
+            .put(ELASTIC_SEARCH_NUMBER_OF_REPLICAS, 0)
+            .build();
         return indexExists()
                    ? updateCurrentIndex(indexSettings)
                    : createNewIndexWithOptimizedSettings(indexSettings);
     }
 
+    public List<BulkResponse> batchInsert(List<IndexDocument> indexDocuments) {
```

Comment:
this is the main change. we break the list of index documents to bulks and send bulks for indexing